nixpkgs_review/cli/__init__.py

```diff
@@ -223,6 +223,13 @@ def common_flags() -> List[CommonFlag]:
             default=read_github_token(),
             help="Github access token (optional if request limit exceeds)",
         ),
+        CommonFlag(
+            "--build-graph",
+            type=str,
+            default=nix_nom_tool(),
+            choices=["nix", "nom"],
+            help='Build graph to use. Use either "nom" or "nix". Will default to "nom" if available',
```

Comment:
```suggestion
            help='Build graph to print. Use either "nom" or "nix". Will default to "nom" if available',
```

Reply:
I like it. It's clearer that it is basically just prints differently